src/components/sequencing/Sequences.svelte

```diff
@@ -78,9 +78,8 @@
           }}
           disabled={workspace === undefined}
           on:click={() => {
-            goto(
-              `${base}/sequencing/new${'?' + SearchParameters.WORKSPACE_ID + '=' + getSearchParameterNumber(SearchParameters.WORKSPACE_ID) ?? ''}`,
-            );
+            const workspaceId = getSearchParameterNumber(SearchParameters.WORKSPACE_ID);
+            goto(`${base}/sequencing/new${workspaceId ? '?' + SearchParameters.WORKSPACE_ID + '=' + workspaceId : ''}`);
```

Comment:
This `workspaceId` check needs to be an explicit check for `null` otherwise a `workspaceId` of `0` will not be included.

Reply:
Can this also be rewritten to 
```
${workspaceId ? `?${SearchParameters.WORKSPACE_ID}=${workspaceId}` : ''}
```
It's a little bit easier to grok than having all the `+` in the string concatenation imo.